cc/blocks/promotion/promotion.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```

Comment:
nit: I wouldn't start with these copyright comments, but if you do add them then the year should be 2023